Author: <gaudenz@debian.org>
Description: add default values and information about keycodes to manpage
--- a/mouseemu.8
+++ b/mouseemu.8
@@ -30,22 +30,45 @@
 .SH OPTIONS
 .TP
 .B -middle B2_MOD B2_KEY
-modifier and key for the middle (second) mouse button
+Modifier and key for the middle (second) mouse button. Defaults to F10 and no modifier
+on PowerPC and Intel Macs, and to none on all other architectures.
 .TP
 .B -right B3_MOD B3_KEY
-modifier and key for the right (third) mouse button
+Modifier and key for the right (third) mouse button. Defaults to F11 and no modifier
+on PowerPC and Intel Macs, and to none on all other architectures.
 .TP
 .B -scroll SCROLL_MOD
-modifier for the scrolling function
+Modifier for the scrolling function. Defaults to Alt.
 .TP
 .B -typing-block DELAY
 Time in milliseconds for which the trackpad will be blocked while typing on the keyboard.
+Defaults to 300ms.
 .TP
 .B -device UINPUT
-device node for the uinput device
+Device node for the uinput device. Defaults to /dev/uinput. If this device is not read and 
+writeable the following devices are also tried: /dev/uinput, /dev/input/uinput and 
+/dev/misc/uinput.
 .TP
 .B -nofork
 don't run in the background
+.TP
+.B -help
+show usage message
+.PP
+The key codes for the buttons and modifiers are key scancodes. They can be found in 
+include/linux/input.h in the kernel headers or by using `showkey` in a console. The 
+keycodes must be given as decimal values (`showkey` displays hex values!).
+
+.SH EXAMPLES
+.PP
+To have the same behaviour as in MacOS X (CTRL-click for right mouse button and no
+emulation for the middle button):
+.PP
+.RS 4
+.B mouseemu -middle 0 0 -right 29 272
+.RE
+.PP
+The code for the (left) mouse button is 272 (0x110 in hex). The code for CTRL is 29. 
 
 .SH AUTHOR
 Mouseemu was written by  Colin Leroy 
